File: Java/프로그래머스/0/120849. 모음 제거/모음 제거.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        for(int i = 0; i < my_string.length(); i++) {
            if(!"aeiou".contains(String.valueOf(my_string.charAt(i)))) {
                answer += my_string.charAt(i);
            }
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120849. 모음 제거/모음 제거.js
function solution(my_string) {
    var answer = '';
    my_string.split('').forEach(item => {
        if(item !== 'a' && item !== 'e' && item !== 'i' && item !== 'o' && item !== 'u') {
            answer += item;
        }
    })
    return answer;
}